internal/liqonet/tunnelEndpointCreator/tunnelEndpointCreator-operator.go

```diff
@@ -457,6 +461,10 @@ func (tec *TunnelEndpointCreator) processLocalNetConfig(netConfig *netv1alpha1.N
 			return nil
 		}
 	}
+	// Store PodCIDR used in remote cluster
+	if err := tec.IPManager.AddPodCIDRPerCluster(netConfig.Status.PodCIDRNAT, netConfig.Spec.ClusterID); err != nil {
```

Comment:
I was wondering: if the AddPodCidrPerCluster operation fails we just continue and log. Is this wanted? Should we retry or handle the error in a different way?

Reply:
Looking at the implementation of AddPodCidrPerCluster we can notice that it can fail only for problems related to communication with the api server: getClusterSubnet and updateClusterSubnets read and write the ipamstorage resource. Retrying seems to be a reasonable way to handle this problem.